src/dmd/target.d

```diff
@@ -328,6 +328,15 @@ struct Target
         }
     }
 
+    /******************
+     * Returns:
+     *  true if xmm usage is supported
+     */
+    extern (C++) bool isXmmSupported()
+    {
+        return global.params.is64bit || global.params.isOSX || global.params.isLinux;
```

Comment:
We don't support OSX32, I don't think theres much point enabling SIMD for it.

Reply:
That's a subject for another PR.